### DISCUSSION TITLE
《独行月球》影评

### DISCUSSION THREAD

**Discussion by giscus** [ZexWoo/Giscus#5] [Announcements]
# 《独行月球》影评

温馨提示：内容可能涉及剧透，如果你计划自己观看，就请不要阅读我的观后感。

https://zexwoo.blog/posts/blogs/moon-man-review/

**Comment by lychichem** [comment]
我看过之后觉得剧情确实可以，想吐槽下模型，既然美国都可以用电影宣传“美利坚科技天下第一”，在我们有长征五号和天宫空间站的今天，我们电影中的火箭模型和空间站模型也完全可以用自家的产品做宣传，而不用弄个土星5号和多年前就有的旋轮空间站模型

**Reply by ZexWoo** [reply]
确实如此，这可以算是个小小的遗憾。
